Atomic/util.py

```diff
@@ -83,7 +83,10 @@ def subp(cmd, cwd=None):
     return ReturnTuple(proc.returncode, stdout=out, stderr=err)
 
 
-def check_call(cmd, env=os.environ, stdin=None, stderr=None, stdout=None):
+def check_call(cmd, env=os.environ, stdin=None, stderr=None, stdout=None, quiet=True):
+    if quiet:
+        devnull = open(os.devnull, 'w')
+        stdout = devnull
     # Make sure cmd is a list
```

Comment:
devnull = open(os.devnull, 'w')


 +        # Show stdout from container if --debug or --verbose
+        standard_out = devnull if not self.args.verbose or self.args.debug else None
+
         # do the scan
-        util.check_call(scan_cmd)
+        util.check_call(scan_cmd, stdout=standard_out)